src/main/resources/db/migration/V6.2.0.1__10288-update_keywordVocabularyURI_metadata.sql

```diff
@@ -0,0 +1,5 @@
+-- update of the "keywordVocabularyURI" metadata to make it consistent with its name "Controlled Vocabulary URL"
```

Comment:
This is obsolete now?

Reply:
The PR description also still mentions changing from URI to URL (not in the code, but might confuse in review or QA)